14/index.php

```diff
@@ -0,0 +1,174 @@
+    <a href="https://community.software.sil.org/t/keyman-14-beta-feedback/4115">Keyman Community site</a> or submit bugs and feature requests to our
+    <a href="https://github.com/keymanapp/keyman/issues/new/choose">Issue Tracker</a></li>
+</ul>
+<br>
+
+<h1 class='red underline'>User Software</h1>
+
+<?php
+downloadSection('Keyman for Windows 14',   'windows',     'keyman-$version.exe', 'beta');
+?>
+
+<h3>What's New in Keyman for Windows 14?</h3>
+
+<ul>
+  <li>Renamed from <strong class="red">Keyman Desktop</strong> to <strong class="red">Keyman for Windows</strong></li>
+  <li>Handles major version upgrades from 10 onward</li>
+  <li>Add GUI for setting configuration options</li>
+</ul>
+
+
+
+<?php
+downloadSection('Keyman 14 for macOS',   'mac',     'keyman-$version.dmg', 'beta');
+?>
+
+<h3>What's New in Keyman 14 for macOS?</h3>
+
+<ul>
+  <li>Added icon for Keyman.app</li>
+</ul>
+
+
+
+<h2 id='linux' class='red underline'>Keyman 14 for Linux</h2>
+
+<h3>What's New in Keyman 14 for Linux?</h3>
+
+<ul>
+  <li>Open a .kmp file with Keyman Config</li>
+  <li>Now supports Ubuntu 20.10 (Groovy)</li>
+  <li>Improved UI</li>
+</ul>
+
+<li>Ubuntu, Wasta-Linux: Keyman for Linux can be installed via launchpad:</li>
+<blockquote><pre class='language-bash code'><code>sudo add-apt-repository ppa:keymanapp/keyman-nightly
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install keyman onboard</code></pre></blockquote>
+
+
+<?php
+downloadSection('Keyman for Android 14', 'android', 'keyman-$version.apk', 'beta');
+?>
+
+<p>You can also <a href=<?= $playstore_signup_link ?>>sign up</a> to access pre-release versions through Google Play.</p>
+<?= $playstoreTable ?>
+
+<h3>What's new in Keyman for Android 14?</h3>
+
+<ul>
+  <li>
+    Select a language during keyboard package installation<br>
+    <img alt='Select language during keyboard installation' src='select_language.png'>
+  </li>
+  <li>Consolidated install menus for installing keyboards</li>
+  <li>Add system globe action to show system keyboards</li>
+</ul>
+
+<h2 class="red underline">Keyman for iPhone and iPad 14</h2>
+<h3>Beta</h3>
+<p>Please register for the pre-release version through the links below for Keyman <a href=<?= $testflight_beta_link ?>>beta</a>
+  pre-releases on your iOS device. This will grant access to the app through <a  href="https://developer.apple.com/testflight/testers/">Apple's
+    Testflight app</a>, which facilitates direct installation on iOS devices.</p>
+
+<?= iosTestFlightTable(); ?>
+
+<h3>What's new in Keyman for iPhone and iPad 14?</h3>
+
+<ul>
+  <li>Select a language during keyboard package installation</li>
+  <li>Improved batching of keyboard and dictionary downloads</li>
+</ul>
+
+
+
+
+
+<?php
+downloadSection('KeymanWeb 14', 'web', 'keymanweb-$version.zip', 'beta');
+?>
+
+<h3>What's New in KeymanWeb 14?</h3>
+
+<ul>
+  <li>Add special keys for right-to-left keyboards</li>
+  <li>Block key previews for blank/hidden keys</li>
+</ul>
```

Comment:
@jahorton do you want anything from this list (you'll need to rewrite the titles from an end-user perspective)?

```
$ grep -E '^\* feat\((web|common/core/web|common/model)' HISTORY.md
* feat(common/core/web): integrated suggestion casing tests (#3887)
* feat(web/engine): allows touch aliasing away from blank keys (#3858)
* feat(common/models): use of applyCasing for suggestions (#3824)
* feat(common/models): casing for suggestions with partial replacement (#3845)
* feat(web/engine): updates osk font, adds layout codes for new glyphs (#3851)
* feat(common/core/web): input processor unit tests (#3836)
* feat(common/core/web): simplify corrective distribution (#3726)
* feat(common/core/web): fat-finger ignores inputs that beep (#3701)
* feat(common/models): context tracking of accepted Suggestions (#3663)
* feat(common/models): context reversion modeling (#3685)
* feat(common/models): disables "keep" when word is not suggestion otherwise (#3700)
* feat(common/core/web): selective wordbreak swallowing after accepting suggestions (#3702)
* feat(common/models): 'revert' now uses model's punctuation (#3647)
* feat(common/models): context + input tracking for predictive text (#3549)
* feat(common/models): initial integration for enhanced corrections (#3555)
* feat(common/models): correction thresholding, acceptance (#3556)
* feat(common/models): Correction improvement (#3575)
* feat(common/models): naive correction-algorithm timer (#3581)
* feat(common/models): core edit-distance calculation class (#3526)
* feat(common/models): Edit path derivation (#3547)
* feat(common/models): low-level correction-algorithm infrastructure (#3527)
* feat(common/models): correction-search algorithm core (#3534)
* feat(common/models): override script defaults: spaces break words (#3506)
* feat(common/models): lexicon traversal (#3479)
* feat(common/models): actual priority queue for Trie models (#3480)
* feat(common/models): efficient batch-enqueue (#3486)
* feat(web): starts proper KMW Sentry integration (#3122)
* feat(web): test Recorder overhaul, Node-based tests using Recorder for KeyboardProcessor (#3060)
* feat(web/engine): Basic KeyboardProcessor tests (#2994)
* feat(common/models/wordbreakers): create join word breaker decorator (#3021)
* feat(web): restyles prediction banner (#2629)
```